skyportal/plot.py

```diff
@@ -159,35 +160,61 @@ def photometry_plot(source_id):
     (str, str)
         Returns (docs_json, render_items) json for the desired plot.
     """
-    color_map = {'ipr': 'yellow', 'rpr': 'red', 'g': 'green'}
+    color_map = {'ipr': 'yellow', 'rpr': 'red', 'g': 'green',
+                 '1': 'green', '2': 'red', '3': 'pink'}
 
     data = pd.read_sql(DBSession()
-                       .query(Photometry, Telescope.nickname.label('telescope'))
-                       .join(Instrument).join(Telescope)
+                       .query(Photometry, Telescope.nickname.label('telescope'), Source.varstar)
+                       .join(Instrument).join(Telescope).join(Source)
                        .filter(Photometry.source_id == source_id)
                        .statement, DBSession().bind)
     if data.empty:
         return None, None, None
 
+    for col in data:
```

Comment:
Are these plotting changes related to this PR?

Reply:
No, these aren't my changes - this is part of what I was referring to when I was saying the merge/rebase looked botched...